spark/v4.0/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestFastForwardBranchProcedure.java

```diff
@@ -165,20 +165,27 @@ public void testInvalidFastForwardBranchCases() {
     assertThatThrownBy(
             () ->
                 sql(
-                    "CALL %s.system.fast_forward('test_table', branch => 'main', to => 'newBranch')",
+                    "CALL %s.system.fast_forward(table => 'test_table', 'main', to => 'newBranch')",
                     catalogName))
         .isInstanceOf(AnalysisException.class)
-        .hasMessage("Named and positional arguments cannot be mixed");
+        .hasMessageStartingWith(
+            "[UNEXPECTED_POSITIONAL_ARGUMENT] Cannot invoke routine `fast_forward` "
```

Comment:
what error do you get if the first arg is a positional one?

Reply:
see https://github.com/apache/iceberg/pull/13106#discussion_r2184173549